Resources/config/routing.xml

```diff
@@ -17,6 +17,12 @@
 
     <services>
 
+        <service id="fos_rest.routing.loader.directory" class="FOS\RestBundle\Routing\Loader\DirectoryRouteLoader">
```

Comment:
At least no good reason :) (we already made the other loader services private in 2.0). I'll change this.